packages/driver/src/cy/commands/querying/querying.ts

```diff
@@ -156,7 +310,8 @@ export default (Commands, Cypress, cy, state) => {
       }
 
       if (aliasObj) {
-        let { subject, alias, command } = aliasObj
+        let { alias, command } = aliasObj
+        let subject = $utils.getSubjectFromChain(aliasObj.subjectChain, cy)
```

Comment:
I was trying to find out where this was used so I pushed a commit throwing an error here: https://github.com/cypress-io/cypress/compare/issue-7306-addSelector-redux...lmiller1990/issue-7306-addSelector-redux?expand=1

Surprisingly only tests related to `type()` broke - is this what you'd expect? Why would this be case?

CI: https://app.circleci.com/pipelines/github/cypress-io/cypress?branch=lmiller1990%2Fissue-7306-addSelector-redux

Reply:
It's because the type() tests invoke `cy.getAll()`, which a custom command registered in `support/utils.ts` and uses `cy.now('get', '@stuff')` - which is the only way to hit this block of code.

Normal users use `cy.get('@foo')` (using the selector), and most of our commands use `cy.get('button')` (which isn't an alias and therefore misses this code block about aliases). Only the combination of aliasing and .now() can end up here, and .type() seems to be the only place in our code base that does so. :woman_shrugging:

Reply:
Hm ok - we will eventually refactor `type` to not go here, then we can delete this block (only used for one thing, which is `type`, which is hard to know by looking at this)?

Reply:
Yeah. This entire block - the whole `add('get')` section - will be removed in my next PR, which I'm currently working on and should have ready today or tomorrow. This is just a messy temporary state because I wanted to limit the scope of things going in at once.